CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
     actually exist.
   * Added application credential support to Assert-M365DSCBlueprint
     FIXES [#1792](https://github.com/microsoft/Microsoft365DSC/pull/1792)
+  * Added checks in New-M365DSCConnection to ensure beta MSGraph profile is correctly set when requested
```

Comment:
Please add this under an UNRELEASED section at the top of the file. You now added it to the v1.23.308.1 release that was released today and this PR is not part of that release.